src/engraving/libmscore/noteentry.cpp

```diff
@@ -92,7 +92,7 @@ NoteVal Score::noteValForPosition(Position pos, AccidentalType at, bool& error)
     }
     case StaffGroup::TAB: {
         if (_is.rest()) {
-            error = true;
+            //error = true;
```

Comment:
is this still needed?

Reply:
This was my question above :-).

As far as I can tell, the only purpose of this line is to prevent anyone from entering rests via mouse on tab staves,  It's been this way basically forever, so mouse entry of rests on tab has not worked in previous releases either.  MU3 even disabled the rest button on the toolbar for tab staves - they *really* didn't want you entering rests onto tab staff by mouse.  But I can't imagine why.

In MU4, the rest button on the toolbar is not disabled, so entry of rests *would* work, if not for this line arbitrarily forbidding it.  Removing this line allows mouse entry of rests to work - both using the rest button on the toolbar, and using right-click - and has no ill effects that I can see.

So I'm happy to simply remove the line rather than comment it out, but I wanted to get feedback on that first.

Reply:
> So I'm happy to simply remove the line

Yes, please